### PR TITLE
Connect ContentContainer to Redux store

### PR DESCRIPTION
**Pull request by Erin007**
Ultimately, we want `Congrats` and all of its children to be connected to the Redux store for isRtl and responsiveSize, in progress here: #19042. `ContentContainer` is a child of `Congrats`, but is also used much more broadly.  This PR connects `ContentContainer` to the Redux store, while maintaining responsive as a prop, and updates imports to `UnconnectedContentContainer`. 

In the future, we may want to access the store for other uses of `ContentContainer`; but, the main motivation for this PR is to reduce the diff in #19042.

### CODE REVIEW

**Comment by caleybrock** [comment]
I think this code change looks good - just have questions about if we want to be using different versions of one component in our code. Let's discuss offline in a bit.

**Comment by caleybrock** [comment]
@islemaster this is the one I wanted your opinion one.

**Comment by islemaster** [comment]
Sorry for the delay on this, looking now.

File: apps/src/templates/ContentContainer.jsx [hunk 1]
```diff
@@ -222,4 +224,9 @@ class Link extends Component {
   }
 }
 
-export default Radium(ContentContainer);
+export const UnconnectedContentContainer = ContentContainer;
+
+export default connect(state => ({
+  responsiveSize: state.responsive.responsiveSize,
+  isRtl: state.isRtl,
+}))(Radium(ContentContainer));
```
Comment (islemaster) [inline apps/src/templates/ContentContainer.jsx]: You probably want `Radium` applied to both the connected and unconnected version of this component.  I recommend the following pattern:
```jsx
export const UnconnectedContentContainer = Radium(ContentContainer);

export default connect(state => ({
  // ...whatever...
}))(UnconnectedContentContainer);
```

File: apps/src/templates/projects/ProjectWidget.jsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React, {PropTypes} from 'react';
 import PersonalRecentProjects from './PersonalRecentProjects.jsx';
-import ContentContainer from '../ContentContainer.jsx';
+import {UnconnectedContentContainer as ContentContainer} from '../ContentContainer';
```
Comment (islemaster) [inline apps/src/templates/projects/ProjectWidget.jsx:3]: It seems preferable to import the connected `ContentContainer` here, and remove the need to pass `isRtl` to it here.  That assumes this component is always used in a connected context though, which may not be the case.

File: apps/test/unit/templates/studioHomepages/TeacherSectionsTest.jsx [hunk 3]
```diff
@@ -4,7 +4,8 @@ import {expect} from '../../../util/configuredChai';
 import {
   UnconnectedTeacherSections as TeacherSections
 } from '@cdo/apps/templates/studioHomepages/TeacherSections';
-import ContentContainer from "@cdo/apps/templates/ContentContainer";
+import { UnconnectedContentContainer  as ContentContainer
+} from "@cdo/apps/templates/ContentContainer";
```
Comment (islemaster) [inline apps/test/unit/templates/studioHomepages/TeacherSectionsTest.jsx:8]: Whitespace turned out funny here. Prefer the pattern used with `UnconnectedTeacherSections` above if this line is too long.

**Comment by caleybrock** [comment]
ooo I like the `ResponsiveContentContainer` idea. I think it makes it more clear what you're trying to do here, but still allows you to do this in stages like you mentioned before.

**Comment by islemaster** [comment]
I totally think you could do that within `ContentContainer.jsx` too, just leave the unconnected one as the default export (since it's the more common use case right now) and you don't need to update all the places it gets used.  Then, if we decide to take the other approach later, it's easy to pivot.

If you expect all `ContentContainer`s to eventually be connected though, I'm fine with the current approach. :+1:

**Comment by Erin007** [comment]
I think eventually all `ContentContainer`s will eventually be connected. As we make more of a concerted effort to support Rtl styling, all of the `ContentContainer` uses will want to connect to the global store to avoid passing isRtl around. I also wouldn't be surprised if we try to responsify more dashboard pages in the future; /congrats and /courses are already responsive and there is now a responsive system we could apply to other pages.

**Comment by islemaster** [comment]
Cool - let's use your current approach then, and gradually convert other ContentContainers over to the connected one as followup work.

**Pull request closed without merging**